acceptance/analyzer_test.go

```diff
@@ -370,21 +386,24 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 
 		when("analyzed path is provided", func() {
 			it("uses the provided analyzed path", func() {
-				execArgs := []string{
-					ctrPath(analyzerPath),
-					"-analyzed", ctrPath("/some-dir/some-analyzed.toml"),
-					fixtures.authRegAppImage,
+				analyzeFlags := []string{"-analyzed", ctrPath("/some-dir/some-analyzed.toml")}
+				if api.MustParse(platformAPI).Compare(api.MustParse("0.7")) >= 0 {
+					analyzeFlags = append(analyzeFlags, []string{"-run-image", fixtures.authRegRunImage}...)
```

Comment:
Is there a reason why you're appending an array instead of appending each element (like you're doing on line 420)?
```
analyzeFlags = append(analyzeFlags, "-run-image", fixtures.authRegRunImage)
```

Reply:
Nope! Thanks for pointing that out.

Reply:
Updated in 07e41be